dynamic_instrumentation/src/instrument_memory.rs

```diff
@@ -293,19 +341,19 @@ impl<'a, 'tcx: 'a> Visitor<'tcx> for CollectFunctionInstrumentationPoints<'a, 't
             .find_instrumentation_def(Symbol::intern("ptr_load"))
             .expect("Could not find pointer load hook");
 
+        let base_ty = self.body.local_decls[place.local].ty;
         // Instrument field projections on raw-ptr places
```

Comment:
Is this comment still accurate? Looks like you now instrument references here too.